orbit/cmd/orbit/orbit.go

```diff
@@ -516,6 +516,11 @@ func main() {
 			return fmt.Errorf("initializing token read writer: %w", err)
 		}
 
+		// ping to the server to ensure we have the latest capabilities
+		if err := orbitClient.Ping(); err != nil {
+			return fmt.Errorf("ping to server: %w", err)
```

Comment:
Just log error? if Fleet is down during the ping then Orbit will fail to start?

Reply:
As I said in the other comment, am ok with removing this ping, the `flagRunner.DoFlagsUpdate()` above already tackles this, maybe just add a comment?